packages/wallets/phantom/src/icons/phantom.svg

```diff
@@ -0,0 +1,31 @@
+<svg width="128" height="128" viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
<svg width="48" height="48" viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg">
```
The issue with the wallet's name comes from here:

<img width="1288" alt="image" src="https://user-images.githubusercontent.com/1245209/229174461-cb69260d-5200-4b52-af3a-78cd3e212559.png">

Sorry, our CSS is not ideal here, and it should be handled better on our side. But for now, let's set the width and height to 48 here.